applications/directory/src/components/CreateProfileForm.js

```diff
@@ -307,6 +307,7 @@ const CreateProfileForm = ({
             '& h3': {
```

Comment:
Need to move `&` styles to the respective elements. No more nested styles here. Time to clean this up.